ginivision/src/main/java/net/gini/android/vision/camera/CameraFragmentImpl.java

```diff
@@ -880,6 +901,20 @@ public void showError(@NonNull final String message, final int duration) {
                 duration).show();
     }
 
+    @Override
+    public void startMultiPage(@NonNull final Document document) {
+        if (!(document instanceof  ImageDocument)) {
+            return;
+        }
+        final ImageDocument imageDocument = (ImageDocument) document;
+        final Photo photo = PhotoFactory.newPhotoFromDocument(imageDocument);
+        // TODO: get rid of bitmap rotation -> rotate only the ImageView in the stack
+        final Bitmap rotatedBitmap = getRotatedBitmap(photo);
```

Comment:
Will be taken care of when we add memory management for bitmaps and jpeg byte arrays.